vsintegration/src/vs/FsPkgs/FSharp.Project/FS/MSBuildUtilities.fs

```diff
@@ -12,6 +12,60 @@ open System.Diagnostics
 open System.IO
 open System
 
+module internal MSBuildHelpers =
```

Comment:
Maybe it's just me, but I am having a very hard time following the logic in here... Maybe some comments with examples and/or more descriptive variable names would help?